src/Microsoft.VisualStudio.ProjectSystem.Managed/Telemetry/IConfiguredProjectLanguageServiceTelemetryService.cs

```diff
@@ -6,26 +6,73 @@
 
 namespace Microsoft.VisualStudio.Telemetry
 {
+    /**
```

Comment:
`/**` has no special meaning in C# (unlike in Java). This could be a `///` style comment instead. A few other instances below too.

Reply:
wow, muscle memory - definitely didn't catch myself doing this :D

---
In reply to: [569849335](https://github.com/dotnet/project-system/pull/6880#discussion_r569849335) [](ancestors = 569849335)